### ISSUE TITLE
[BUG] auto not working on google chrome

### ISSUE THREAD

**Issue by phseiff** [darkreader/darkreader#2722]
**Describe the bug**
DarkReader.auto does not seem to work on google chrome, even though it behaves as expected on firefox (both desktop)

**To Reproduce**
I use Google Chrome, Version 83.0.4103.61 (Official Build) snap (64-bit), on Ubuntu 18.04, and [this](phseiff.github.io) website is shown in light mode even though my system, as well as my browser, use dark mode. On firefox, it works as expected.

**System Information:**
 - OS: Ubuntu 18.04 with Gnome
 - Browser: Google Chrome
 - Version: 83.0.4103.61 (Official Build) snap (64-bit)

**Comment by Gusted** [comment]
Which automation mode do you use?

**Comment by phseiff** [comment]
I took [darkreader.js](https://cdn.jsdelivr.net/npm/darkreader@4.9.10/darkreader.js) from jsDelivr (as suggested by README.md), embedded it into my page (not without including the license, of cause), and used

```DarkReader.auto({
    brightness: 100,
    contrast: 90,
    sepia: 10
});
```

as suggested by README.md.
And I used `DarkReader.setFetchMethod(window.fetch)` because it helped to fix some issues on firefox. The dark mode is applied whenever my os uses dark mode (on firefox), but not on chrome.

Is that what you meant when you asked which automation mode I use? If not, can you give me further instructions to find out which automation mode I am using?

**Comment by Gusted** [comment]
Auto mode will bassicly watch for the user's `(prefers-color-scheme: dark)`. Dark -> Enabled | Light -> Disabled

**Comment by phseiff** [comment]
Does this mean that the user's preferred color scheme can't be accessed on Chrome?

**Comment by Gusted** [comment]
`matchMedia('(prefers-color-scheme: dark)').matches;` What does this return when you run it into your dev tools console?

**Comment by phseiff** [comment]
`false`

**Comment by Gusted** [comment]
Hmm than I'm sorry to say that this could be a browser bug as we just use this API as this is the only way to take a `dark mode` option from the system. I suggest to search with Ubuntu Gnome and dark mode Chrome etc. If you don't find any I suggest opening a chromium bug report about it.

**Comment by phseiff** [comment]
That's a relief to hear since most people don't use this combination of os and browser, I guess. Thanks for your help in figuring it out!

**Comment by Gusted** [comment]
If you like to you can post updates and share the chromium bug. If you need any more info for it I'm happy to hear and help with it.

**Comment by phseiff** [comment]
I will keep you up to date about it. I now searched for any related bug on bugs.chromium.org, and there are only 6 bugs related to prefers-color-scheme at all, so I just issued a bug report there. I'll be automatically informed via email once there is an answer, and I'll keep you up to date about it here.

**Comment by Gusted** [comment]
If you don't mind to share the link so I can sign up for notifications as well :)

**Comment by phseiff** [comment]
Yes, here it is: (https://bugs.chromium.org/p/chromium/issues/detail?id=1091468)[https://bugs.chromium.org/p/chromium/issues/detail?id=1091468]. Though I'm not sure if it is going to bear fruits; I don't have any experience in filing in chrome bug reports.

**Comment by Gusted** [comment]
I'm my eyes a good bug report, just hope the developers / bug handlers want to download a new  OS that's the only potential problem they wont respond.

**Comment by phseiff** [comment]
If the problem can be reproduced on any Ubuntu 18.04 with GNOME, it shouldn't be that much of a deal (after all, they seem to have lots of Linux-specific issues, and Linux tends to be split into different distributions), and if it is really specific to ZORIN OS, it shouldn't be a problem if they don't fix it since ZORIN OS comes with firefox as the default browser, and most people won't use Chromium on a Linux with preinstalled firefox. But I'm pretty sure they'll at least try to reproduce it using Ubuntu and gnome, if it isn't a duplicate.

**Comment by 1ncend1ary** [comment]
For anyone looking for a temporary fix: make a script

```
win=$(xdotool search --class chromium | tail -1)
xdotool windowactivate --sync $win
xdotool key --clearmodifiers Alt+Shift+d
```
and run it every time you toggle dark mode (which for me is already a script that I run). Remember to install `xdotool`.

The script finds any open chromium window, switches to it and then inputs keys to toggle the `darkreader` extension. Mind that it could theoretically fail if it found an incorrect window id (to find all: `xdotool search --class chromium`) and it just toggles dark mode, you'd need to keep track of the current mode in a file and write a conditional expression to fix that.

Theoretically, you could input keys without switching to the chromium window, but I couldn't make it work. 

Source: https://bbs.archlinux.org/viewtopic.php?pid=963630#p963630

**Comment by lockieluke** [comment]
Does not work on Mac

**Comment by Stooovie** [comment]
No auto-toggle mode is working for me on Mac (Sonoma 14.1.1) and Arc browser. Location for Arc is enabled.

**Comment by joelsleeba** [comment]
Chromium 119.0.6045.159 Arch Linux

Same error
